content/docs/welcome-guide/setup/setup-from-github/building-linux.md

```diff
@@ -0,0 +1,175 @@
+* Configured the required Linux software as described in the [Linux software configuration](/docs/welcome-guide/requirements/#linux) section of the system requirements topic.
+
+## Build the engine
+
+To prepare to build the engine and projects, choose one of the following build types based on the primary focus of your development work, then follow the instructions in the corresponding tab:
+
+1. **Source engine** - Choose this build type if you plan to make frequent changes to the engine source code.
+
+1. **Pre-built SDK engine** - Choose this build type if you're primarily interested in project development and you plan to make only infrequent changes (or no changes) to the engine source.
+
+{{< tabs name="Engine build instructions" >}}
+{{% tab name="Source engine" %}}
+
+1. Create a package directory in a writeable location. The following examples use the directory `$HOME/o3de-packages`.
+
+    ```shell
+    mkdir $HOME/o3de-packages
+    ```
+
+    The O3DE package downloader uses this directory to retrieve external libraries needed for the engine.
+
+1. Get the Python runtime, which isn't included in the GitHub repo. The `o3de` script (part of the **O3DE CLI**) requires this runtime. You'll use this script to run common command line functions. This script also requires **CMake** to be installed and accessible on your device's path. If you haven't installed CMake, or you get an error that CMake cannot be found when running the script, refer to the [O3DE System Requirements](/docs/welcome-guide/requirements) page for installation instructions.
+
+    Open a terminal window and change to the directory where you set up O3DE, then run the `get_python` script.
+
+    ```shell
+    python/get_python.sh
+    ```
+
+1. Use CMake to create the Linux build project for the engine. Supply the build directory, the Ninja Multi-Config generator, the path to the packages directory that you created, and any other project options. Paths can be absolute or relative. Alternatively, you can use the CMake GUI to complete this step.
+
+    ```shell
+    cmake -B build/linux -S . -G "Ninja Multi-Config" -DLY_3RDPARTY_PATH=$HOME/o3de-packages
+    ```
+
+    The preceding command specifies several noteworthy custom definitions (`-D`). All are optional but recommended in this example.
+
+    * `LY_3RDPARTY_PATH` : The path to the downloadable package directory, also known as the "third-party path". Do not use trailing slashes when specifying the path to the packages directory.
+
+1. (Optional) Use CMake to build the source engine. This step is optional because in the "source engine" build model, the engine is built inside of every project. If you plan on working with projects, to avoid building the engine twice, consider waiting until you learn how to create and build a project, which we cover in our documentation on creating projects. The following command builds the engine without a project.
```

Comment:
Can you link to docs on creating projects?

Reply:
I would rather not give them a link here, actually. I don't want to tempt them to jump out here, because at this point, the reader should really complete the rest of the steps on this page. There's a link at the end.